### DISCUSSION TITLE
GIF Compilation

### DISCUSSION THREAD

**Discussion by GalexY727** [Patribots4738/Crescendo2024#268] [Show and tell]
This should be a discussion chain of all of the gifs of things we have done with the bot so that there can be one simple place for anyone to see.

**Comment by tina-hague** [comment]
We already have one in the readme

**Reply by GalexY727** [reply]
This discussion is mainly for our design process, not necessarily what we have now. For example, the creation of SWD and chain intercept calculations

**Comment by GalexY727** [comment]
![swd1](https://cdn.discordapp.com/attachments/951910178968731648/1205753119145861180/AdvantageScope_JPjNv0DlU1.gif?ex=6607a899&is=65f53399&hm=a6968bb5d86cbe59deecd9dd3c802d48ae24346ac1dd0f3cafd3f9e5d7b33b47&)
![swd2](https://cdn.discordapp.com/attachments/1043438899567001621/1205762053554446376/AdvantageScope_2y1Vvow8Yg.gif?ex=6607b0eb&is=65f53beb&hm=4397203183f638cbef13c8130321d0a37f8707982463b621189abd26a2822a7a&)
![swd3](https://github.com/Patribots4738/Crescendo2024/assets/65139378/6d5c357b-a4cf-44ce-9d83-bc2f0258a912)
![swd4](https://github.com/Patribots4738/Crescendo2024/assets/65139378/1ea5ba65-9d2b-4a2d-93cb-2a7f0324e0a4)
![earlyswd](https://github.com/Patribots4738/Crescendo2024/assets/65139378/0b34f050-4c1d-4c6a-999a-9103f0df5131)

**Comment by GalexY727** [comment]
![chain1](https://github.com/Patribots4738/Crescendo2024/assets/65139378/74e8e04e-7a0f-4e0a-8e83-a06f9a556713)
![chain2](https://github.com/Patribots4738/Crescendo2024/assets/65139378/3746657e-59b4-4c10-bf3b-70ebb4ea264e)

**Comment by GalexY727** [comment]
![notetrajectorydemo](https://github.com/Patribots4738/Crescendo2024/assets/65139378/485a4d6c-8512-443f-a3a1-694596f4e217)

**Comment by GalexY727** [comment]
![simcam](https://github.com/Patribots4738/Crescendo2024/assets/65139378/7d8ecf80-4ca6-4d8c-b5dd-4ee48e26bdbc)

**Comment by GalexY727** [comment]
![simulaedleds](https://github.com/Patribots4738/Crescendo2024/assets/65139378/46078daa-6a98-4d9b-8632-a2ecc171efb5)

**Comment by GalexY727** [comment]
![4414-pass](https://github.com/Patribots4738/Crescendo2024/assets/65139378/0680ca7c-82ec-48ae-89b4-579f5338a8d2)

**Comment by tina-hague** [comment]
could we add our gifs to the images folder cause git don't consistently load
